plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -836,6 +837,7 @@ public boolean isTableCurrent(ConnectorSession session, ConnectorTableHandle tab
 
     @Override
     public MaterializedViewFreshness getMaterializedViewFreshness(ConnectorSession session, SchemaTableName materializedViewName)
+            throws MaterializedViewNotFoundException
```

Comment:
this is redundant, but please add `@throws MaterializedViewNotFoundException` in `io.trino.spi.connector.ConnectorMetadata#getMaterializedViewFreshness` javadoc.

Reply:
Actually, `MaterializedViewNotFoundException` is runtime exception so you don't have to add it here (and in `io.trino.spi.connector.ConnectorMetadata#getMaterializedViewFreshness`).
In fact, I don't see we mark methods with throws `TableNotFoundException` or `ViewNotFoundException` too.

Wdyt @findepi?

Reply:
> I don't see we mark methods with throws

that's why i said to add `@throws` to javadoc.

Reply:
remove `throws MaterializedViewNotFoundException`